### DISCUSSION TITLE
virtual desktops problems

### DISCUSSION THREAD

**Discussion by dabeegmon** [lxqt/lxqt#2269] [General]
### already listed in bugs - - - sorry for the noise

Greetings

Have been running lxqt for a long time - - - love it - - - thanks a muchly for the work. 
See that there is an attempt to move to version 0.17. 
is there going to be a version 1.0 sometime?

I'm having problems with 2 things and they are likely connected but dunno. 
(I'm very good at using things real hard so I tend to find breakage points!!!) 

When I set the names to the workspaces (both the one found in Preferences/Workspaces and by clicking on the workspace switcher on the panel) I need to rewrite the labels after every reboot. 
How do I make those labels more permanent?

TIA

**Comment by stefonarch** [comment]
Are you using xfwm4? In fact this is not resolved: https://github.com/lxqt/lxqt-panel/issues/1693
Personally I prefer kwin. Every WM has it's downsides unfortunately.

>See that there is an attempt to move to version 0.17.
is there going to be a version 1.0 sometime?

Debian or debian based I assume. Very soon LXQt 1.1 should be in testing https://twitter.com/lxqt_project/status/1547471595639291904

>At about the same time next week, 1.1 should land in Debian Sid, making it to Testing sometime before the end of the month

**Reply by dabeegmon** [reply]
Apologies - - - - I did search the discussions but thought that a discussion would have happened before a bug was filed - - - my mistake. 

I find myself wishing that that limit of 32 desktops also isn't hard (I've got more than that in use at present). 
Looking forward to the new version. 

Hmmmm - - - - as I don't use twitter that is a welcome announcement. 

I understand software is very tough to pin down but - - - any idea on a ETA?

**Reply by stefonarch** [reply]
Nothing to apologize :)

If you mean ETA for the issue to be fixed: no idea. Of the 3 WM mainly used xfwm4 is the only one where desktop names cannot be configured (it seems possible only from inside a XFCE session). So maybe it cannot be solved.

**Reply by dabeegmon** [reply]
Hmmmmm - - - went looking. 

Can't find anything besides your link re: lxqt 1.1 and what I'm finding there seems to be related to Win10. 

Any chance that you might know of other places to look?

TIA

**Reply by stefonarch** [reply]
As I said: I once tried even configuring desktop names under XFCE session, logging back into LXQt those settings were gone. Didn't investigate further, this was one more reason to adapt kwin for me.

**Reply by edwardp3** [reply]
> Debian or debian based I assume. Very soon LXQt 1.1 should be in testing https://twitter.com/lxqt_project/status/1547471595639291904
> 
> > At about the same time next week, 1.1 should land in Debian Sid, making it to Testing sometime before the end of the month

I have Debian Sid and Testing installed in virtual machines. Looking forward to testing it.
